### ISSUE TITLE
Replacement for this DEAD PROJECT?

### ISSUE THREAD

**Issue by navid-zamani** [SufficientlySecure/document-viewer#344]
Since this is abandoned, I’m asking eveyone who comes here, what they use instead…

It has to open PDF, DjVu, EPUP, XPS, and must not be ”minimalistic“ or in other ways over-”simplified“. And preferably open source, but in any case *not* nagware/adware/spyware or other commercial malware.

Here’s the [obligatory AlternativeTo.net](https://alternativeto.net/software/ebookdroid/?feature=ebook-reader&license=opensource&platform=android) link.
Seems like Librera Reader has the same security holes. And Shelf isn’t in F-Droid.

**Comment by navid-zamani** [comment]
I’ve used KOReader now, and it seems OK, even if quite e-reader optimized.

**Comment by ildar** [comment]
asking here isn't the best idea.
It's worth looking at the F-Droid site.
One possible alternative is LibreraReader BTW.

**Comment by navid-zamani** [comment]
@ildar: You literally didn’t even read what I wrote.

**Comment by ildar** [comment]
Yeah, sorry.
I totally missed everything under "Alternatives.net" link. Dunno how it
happened.
Anyway asking about new apps is better at e.g. F-Droid forum, not the dead
project's issue tracker.

**Comment by navid-zamani** [comment]
@ildar: Yes, you are right. I just thought this bug would serve well to lead people who end up here to wherever that info was found. So your suggestion of the F-Droid forums is perfect.

In any case, I only leave this bug open as a canary, to see if anyone with maintainer rights is still alive to close it. :) (Feel free)

**Comment by woj-tek** [comment]
> Anyway asking about new apps is better at e.g. F-Droid forum, not the dead
> project's issue tracker.

Actually I just saw notification about vulnerability in f-droid and though to check out the repository and stumbled on this issue so... there is definitely certain path leading people here :-)

I checked LibreraReader and it seems quite feature-rich... (too much to my teste). The next one I tried was MJ PDF and it seems kinda no-frill reader.

**Comment by 6543** [comment]
@SpiritQuaddicted well would you support it, if I would create an organization, to move this project there and let it maintain by me and others who do want to see this project maintained to be used daily?

**Comment by SpiritQuaddicted** [comment]
What do I have to do with that? I think you tagged me by accident.

**Comment by 6543** [comment]
hahah rigt it shold be @SufficientlySecure ...

**Comment by 6543** [comment]
or @dschuermann ?
